app/assets/stylesheets/custom.scss

```diff
@@ -321,3 +321,25 @@ ul, menu, dir {
     width: 100px;
     height: 30px;
 }
+
+#flash_notice {
+  &.error {
+    border: solid 4px #900;
+          background-color: #FF6565;
```

Comment:
Color `#FF6565` should be written as `#ff6565`<br>Line should be indented 4 spaces, but was indented 10 spaces